other/install_2-1_mysql.sql

```diff
@@ -90,8 +90,8 @@ CREATE TABLE {$db_prefix}ban_groups (
 CREATE TABLE {$db_prefix}ban_items (
   id_ban MEDIUMINT UNSIGNED AUTO_INCREMENT,
   id_ban_group SMALLINT UNSIGNED NOT NULL DEFAULT '0',
-  ip_low VARBINARY(16),
-  ip_high VARBINARY(16),
+  ip_low VARBINARY(16) DEFAULT NULL,
```

Comment:
a null able field is by sql def default null -> so no need to write this down.

Reply:
OK I'll change all of these.   Some had it, some didn't, all I did was make it consistent one way or the other.  I'll remove.